src/main/java/com/exactpro/th2/conn/MicroserviceMain.java

```diff
@@ -162,7 +163,12 @@ public static void main(String[] args) {
                     EventType.SERVICE_EVENT, serviceEventsRoot.getId()
             ));
 
-            IServiceListener serviceListener = new ServiceListener(directionToSequence, configuration.getSessionAlias(), processor, eventDispatcher);
+            IServiceListener serviceListener = new ServiceListener(
+                    getActualSequences(),
+                    factory.newMessageIDBuilder(),
```

Comment:
I'm not sure about the idea to reuse one builder to create several message ids. Concurrent or logical problems can happen. Let's pass a supplier to produce new builder

Reply:
```suggestion
                    factory::newMessageIDBuilder()
```

Reply:
done, and also I forgot about `ConnectivityConfiguration .sessionAlias` in my first commit, please see

Reply:
done